appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java

```diff
@@ -1757,7 +1762,7 @@ public void addResourceAdapter(String raName){
      * @return the Set of application paramaters.
      */
     public Set<ApplicationParam> getApplicationParams() {
-        return applicationParams; 
+        return applicationParams;
     }
 
    /**
```

Comment:
This file just seems to have a load of white space changes only is that correct?